keymaps/git.cson

```diff
@@ -3,6 +3,19 @@
   'ctrl-shift-8': 'github:toggle-github-tab'
   'ctrl-9': 'github:toggle-git-tab-focus'
   'ctrl-shift-9': 'github:toggle-git-tab'
+  'alt-g enter': 'github:commit'
+  'alt-g f': 'github:fetch'
+  'alt-g F': 'github:pull'
```

Comment:
@damieng Sadly those ones are taken by [git-diff](https://github.com/atom/git-diff/blob/master/keymaps/git-diff.cson). I kind of wanted those too, to be honest :smile: